lxd/db/containers.go

```diff
@@ -241,22 +241,35 @@ SELECT nodes.id, nodes.address
 // string, to distinguish it from remote nodes.
 //
 // Containers whose node is down are addeded to the special address "0.0.0.0".
-func (c *ClusterTx) ContainersListByNodeAddress(project string) (map[string][]string, error) {
+func (c *ClusterTx) ContainersListByNodeAddress(project string, instanceType instance.Type) (map[string][]string, error) {
```

Comment:
The function itself will need renaming eventually, but we can do one big `sed` commit later once we're done with the more critical stuff